doc/contributor/svm.md

```diff
@@ -86,5 +81,34 @@ Hello
 0.25 66824
 ```
```

Comment:
Should the whole document be in contributors? I think startup numbers and "how to use" could be part of user documentation.

Reply:
How to use is easy - it's the default. Actually using the SVM doesn't need to be documented for users as we wouldn't normally expect them to do that. The numbers are useful but only if you're talking about the implementation and results and things and that's more of an internal thing.

Reply:
OK, maybe we should just link from the README when talking about startup to this table.
Although I think the README already says like MRI which might be enough.

Reply:
Fixing.